src/components/Interface/AddBook.module.css

```diff
@@ -1,3 +1,27 @@
-body {
-  background-color: #fff;
+.addbook {
+  border-top: 1px solid #e8e8e8;
+  padding-top: 15px;
+}
+
```

Comment:
- [ ] Please style your dropdown to select the category to look like the original design and the color of your `ADD NEW BOOK` title

Your design 👇 

![image](https://user-images.githubusercontent.com/75126481/174143562-7c3b1f9a-fff1-43bc-afe3-d4a0cc3d11e2.png)

Original design 👇 

![image](https://user-images.githubusercontent.com/75126481/174143975-c1fa75d5-5cea-424f-8ca0-9c6990faa25a.png)